discovery-provider/src/tasks/playlists.py

```diff
@@ -173,7 +180,7 @@ def parse_playlist_event(
 
     if event_type == playlist_event_types_lookup["playlist_tracks_ordered"]:
         if getattr(playlist_record, 'playlist_contents') is not None:
-            logger.warning('playlist event playlist_tracks_ordered')
+            logger.info('playlists.py | Processing playlist event: [playlist_tracks_ordered]')
```

Comment:
@vicky-g instead of just spitting out a string with `[someevent]` is there any relevant information here that we can log?

Reply:
oooh ok like something unique pertaining to the playlist. let me add something!